configs/mac/mac_clevr.yaml

```diff
@@ -0,0 +1,13 @@
+# Load the CLEVR config first.
+default_configs:
+    configs/mac/default_clevr.yaml
+
+# Add the model parameters:
+model:
```

Comment:
Maybe we could move model to default_mac.yaml as well?

Reply:
Given that these experiments are on 2 models (MAC & S-MAC), not sure it's very useful